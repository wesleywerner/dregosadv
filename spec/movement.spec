brief
go faster
exit
north
west
go up
go down
climb tree
climb down tree
climb down
go northeast
go up
go down
climb stairs
cross bridge
jump over waterfall
