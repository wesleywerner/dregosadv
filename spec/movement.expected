   The Maro River                                 Time:  3:00 pm
  
  April 10th, 1940.
  
  Your journey started a few days ago after docking at Port Moresby, from
  there it was two day's ride on the back of a truck to Kerema, followed by a
  small aircraft flight and landing near the mouth of the Maro River.
  
  You met your guide and his canoe, who will take you up-river to your final
  destination -- a remote and relatively uncharted area in New Guinea -- to
  find the legendary Golden Egg of Man-Toomba
  
  Dr Ego and the Egg of Man-Toomba
  An Interactive Fiction Story
  by Special Agent.
  Release 1 / Serial number 200410 / Inform v6.33 Library v6.12.3pre SX
  
  The Maro River (in the canoe)
  You are in a canoe on the Maro river. Your guide is with you, rowing the
  craft.
. 
> >   The Maro River                                 Time:  3:00 pm
. 
  Dr Ego and the Egg of Man-Toomba is now in its "superbrief" mode, which
  always gives short descriptions of locations (even if you haven't been
  there before).
  
> >   River Shore                                    Time:  3:01 pm
. 
  You hasten your journey.
  
  "Here we are", your guide says -- the canoe comes to a stop on the shore.
  
  River Shore (in the canoe)
. 
> >   River Shore                                    Time:  3:02 pm
. 
  You get out of the canoe.
  
  River Shore
  
  Your guide sits in the canoe gazing out over the moving river, the salty
  water gently lapping the sides of his craft.
  
> >   Jungle Path                                    Time:  3:03 pm
. 
  Jungle Path
  
> >   In a Clearing                                  Time:  3:04 pm
. 
  In a Clearing
  
> >   Top of Tree                                    Time:  3:05 pm
. 
  You climb up the strong branches of the fig tree.
  
  Top of Tree
  
> >   In a Clearing                                  Time:  3:06 pm
. 
  In a Clearing
  
> >   Top of Tree                                    Time:  3:07 pm
. 
  You climb up the strong branches of the fig tree.
  
  Top of Tree
  
> >   Top of Tree                                    Time:  3:08 pm
. 
  (To go up or down the fig tree simply type "go up" or "go down". To climb
  the fig tree type "climb fig tree")
  
> >   In a Clearing                                  Time:  3:09 pm
. 
  In a Clearing
  
> >   Stone Stairs                                   Time:  3:10 pm
. 
  Stone Stairs
  
> >   Rope Bridge                                    Time:  3:11 pm
. 
  Rope Bridge
  
> >   Stone Stairs                                   Time:  3:12 pm
. 
  Stone Stairs
  
> >   Rope Bridge                                    Time:  3:13 pm
. 
  You ascend the stone stairs.
  
  Rope Bridge
  
> >   Waterfall                                      Time:  3:14 pm
. 
  Waterfall
  
> >   Plunge Pool                                    Time:  3:15 pm
. 
  You take a breath and jump over the edge of the waterfall...
  
  Plunge Pool
  
  There is an ancient temple here.
  
> >   Temple                                         Time:  3:16 pm
. 
  Temple
  
> >   Plunge Pool                                    Time:  3:17 pm
. 
  Plunge Pool
  
  There is an ancient temple here.
  
> >   Temple                                         Time:  3:18 pm
. 
  Temple
  
> >   Plunge Pool                                    Time:  3:19 pm
. 
  Plunge Pool
  
  There is an ancient temple here.
  
> >   Temple                                         Time:  3:20 pm
. 
  Temple
  
> >   Plunge Pool                                    Time:  3:21 pm
. 
  Plunge Pool
  
  There is an ancient temple here.
  
> >